src/test/rgw/rgw_multi/tests_ps.py

```diff
@@ -2221,6 +2222,99 @@ def test_ps_subscription():
     topic_conf.del_config()
     master_zone.delete_bucket(bucket_name)
 
+
+def test_ps_admin():
+    """ test radosgw-admin commands """
+    master_zone, ps_zone = init_env()
+    bucket_name = gen_bucket_name()
+    topic_name = bucket_name+TOPIC_SUFFIX
+    realm = get_realm()
+    zonegroup = realm.master_zonegroup()
+
+    # create topic
+    topic_conf = PSTopic(ps_zone.conn, topic_name)
+    topic_conf.set_config()
+    result, status = ps_zone.zone.cluster.admin(['topic', 'list', '--uid', get_user()])
```

Comment:
this is missing the --rgw-zone argument. in teuthology, this pubsub zone (test-zone4) is running on the same cluster as test-zone2, and test-zone2 is set as the --default

the function `Zone.zone_arg()` returns `['--rgw-zone', self.name]`, so you can add that to your commands:
```python
cluster.admin(['topic', 'list', '--uid', get_user()] + ps_zone.zone.zone_arg())
```